src/components/header/Header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.videoWrap {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.videoWrap video {
+  min-width: 100%;
+  min-height: 100%;
+}
+
+.headerOverlay {
+  height: 100vh;
+  width: 100vw;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: #2c3547;
+  z-index: 1;
+  opacity: 0.85;
+}
+
+.headerContent {
+  z-index: 2;
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: repeat(12, 1fr);
+}
+
+.line {
+  width: 70%;
+  height: 2px;
+  border: 1px solid white;
+}
+
+.lineWrapper {
+  grid-column: 1/2;
+  grid-row: 5/6;
+  @media (max-width: 968px) {
+    display: none;
+  }
+}
+
+.content {
```

Comment:
need a better classname than `.content` or `.line` because it should easily clash